src/tests/plans/D3MCompoundPlan.t.sol

```diff
@@ -0,0 +1,350 @@
+// SPDX-FileCopyrightText: © 2022 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity ^0.8.14;
+
+import {DSSTest} from "dss-test/DSSTest.sol";
+import "../interfaces/interfaces.sol";
+
+import { D3MPlanBaseTest } from "./D3MPlanBase.t.sol";
+import { D3MCompoundPlan } from "../../plans/D3MCompoundPlan.sol";
+
+interface CErc20Like {
+    function borrowRatePerBlock()               external view returns (uint256);
```

Comment:
Same suggestion as per #76 (for compound pool and plan interfaces)

Reply:
see https://github.com/makerdao/dss-direct-deposit/pull/78